client/sidebar.jsx

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import axios from 'axios';
+import Style from './style.jsx';
 import Confidence from './img/Confidence.png';
 import ShipQuickly from './img/ShipQuickly.png';
 import QuickerShipperImg from './img/QuickShipper.png';
 
+const PATH = document.location.pathname.substring(1);
+
 class Sidebar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      product: props.product,
-      seller: props.seller,
+      product: {},
+      seller: { reviews: { rating: 0 } },
       isWatched: false
     };
   }
 
   // when initializing the page
   componentDidMount() {
+    console.log(PATH);
     // send a get request for the product and seller infomation
-    axios.get('http://localhost:3210/item/0')
+    axios.get('http://localhost:3210/api/item/0')

```

Comment:
We are still getting the first item in the database. Should we get the right item depending on what the URL path name is?

Reply:
I can't put the dynamic path in it without having problem, so I'm still trying to figure out how to do it.

Reply:
got it! I'll approve this and we can resolve that in a separate PR then.